jeandle-docs/getting-started.md

```diff
@@ -6,6 +6,12 @@ For detailed guidance on getting started with upstream LLVM and OpenJDK, refer t
 + LLVM: [https://llvm.org/docs/GettingStarted.html](https://llvm.org/docs/GettingStarted.html)
 + OpenJDK: [https://openjdk.org/guide/](https://openjdk.org/guide/)
 
+To streamline environment configuration, a Dockerfile is provided. The following commands shows how to build a container image named jeandle-dev:latest.
+```
+cd jeandle-jdk
+docker build -t jeandle-dev:latest -f Dockerfile .
```

Comment:
It's better to contain a complete guidance, including how to deliver container and how to compile jeandle-llvm and jeandle-jdk